package.json

```diff
@@ -2,7 +2,7 @@
   "name": "react-flask-app",
   "version": "0.1.0",
   "private": true,
-  "proxy": "http://localhost:5000",
+  "proxy": "http://127.0.0.1:5000",
```

Comment:
Why was this changed? 🤔 Most likely I don't see the full picture. I would think that we want to run our things on localhost when developing.

Reply:
True, I changed it back to localhost.

Reply:
Interesting 🤔 Maybe this is needed by the local flask BE?
But no worries, I was just wondering about this change.